### PR TITLE
Srivastava2014dropout+darting melody

### PR DESCRIPTION
**Pull request by DartingMelody**

### CODE REVIEW

File: _summaries/srivastava2014dropout_2.md [hunk 1]
```diff
@@ -1,19 +1,55 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: srivastava2014dropout
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: DartingMelody
+score: 10/10
```
Comment (zhaoyue-zephyrus) [inline _summaries/srivastava2014dropout_2.md:7]: Only 10 is fine (`/10` will be automatically generated)

File: _summaries/srivastava2014dropout_2.md [hunk 2]
```diff
@@ -1,19 +1,55 @@
...
+2. It approximates to combining various different architectures by sampling from an exponential set of architectures. 
+3. It adds noise that allows the optimization process to explore different regions of weight space.
+4. It reduces co-adaptations by making other hidden units unreliable. Therefore, any unit should become self robust in different contexts and does not rely on          particular units to rectify their mistakes. This can be seen in the following figure.
+5. The Activation of hidden units become sparse. 
+
+![Without and with dropout first layer features](./srivastava2014dropout_2b.png)
+
+**How is it realized (technically)?**
+
+The dropout is realised by the following feed forward equations (* denotes element wise product) :
```
Comment (zhaoyue-zephyrus) [inline _summaries/srivastava2014dropout_2.md:27]: What position is dropout normally placed in?

**Comment by zhaoyue-zephyrus** [comment]
Great job! The summary nicely describes the core idea behind dropout and its features. I've left some minor comments.  Besides, think about the following question: the implementation in the paper is as it is, however, can you spot any potential drawback of the proposed implementation and can you come up with some ideas for remedy? (you don't need to have the answer and write it down)

File: _summaries/srivastava2014dropout_2.md (hunk 2 above)
Comment (DartingMelody) [reply _summaries/srivastava2014dropout_2.md]: Hi Yue @zhaoyue-zephyrus , I have updated the PR. Thanks for the review and for giving ideas to look further into dropout.

**Pull request merged**